application/src/main/java/org/togetherjava/tjbot/features/jshell/JShellCommand.java

```diff
@@ -167,7 +166,8 @@ private void handleEval(IReplyCallback replyCallback, @Nullable User user, boole
     private void handleSnippetsCommand(SlashCommandInteractionEvent event) {
         event.deferReply().queue(interactionHook -> {
             OptionMapping userOption = event.getOption(USER_PARAMETER);
-            User user = userOption == null ? event.getUser() : userOption.getAsUser();
+            Member user = Objects
```

Comment:
You have this everywhere `Member user`, seems more intuitive to go with `Member member`. I'm guessing it's just leftovers from `User` -> `Member` type migration.